zetaclient/config/config_chain.go

```diff
@@ -59,6 +60,13 @@ func bitcoinConfigRegnet() BTCConfig {
 	}
 }
 
+// solanaConfigLocalnet contains config for Solana localnet
+func solanaConfigLocalnet() SolanaConfig {
+	return SolanaConfig{
+		Endpoint: "http://solana:8899",
```

Comment:
nit: isn't this hardcoded somewhere else? Probably we can get this value out of an env variable or config, so we can change it without the need of recompiling.

Reply:
The file `config_chain.go` contains the values for local E2E networks (btc, evm, solana).